### DISCUSSION TITLE
Add flag to disable keyboard increment and decrement functionality in NumberInput

### DISCUSSION THREAD

**Discussion by Sajarin-M** [mantinedev/mantine#5933] [Feature requests]
Is it possible to add a flag to the NumberInput component to disable the keyboard increment and decrement functionality?
I am ready to create a pull request if this is accepted.

**Comment by rtivital** [comment]
Why you need that?

**Reply by Sajarin-M** [reply]
I use multiple number inputs in a grid.
When I click the arrow down, I want to move focus to the input below, and similarly for the arrow up. 
So, I would like to disable the increment/decrement feature while keeping the other useful features of the NumberInput, such as incrementing/decrementing via controls, setting minimum and maximum values, formatting, etc.

I just thought to discuss it. maybe this is just a specific case for me.

**Reply by Sajarin-M** [reply]
Fun fact: Currently, I throw an error from onKeyDown just to get this 🤯

**Comment by Sajarin-M** [comment]
Can this be done? 👀
